packages/client/package.json

```diff
@@ -37,15 +39,14 @@
     "@types/stripe-v2": "^2.0.1",
     "babel-plugin-relay": "^12.0.0",
     "eslint": "^8.2.0",
-    "eslint-config-prettier": "^6.15.0",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-emotion": "^10.0.14",
-    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
this plugin converted all of prettiers rules to slower eslint rules. it's faster to run a bare minimum of eslint rules & then run prettier after that